SuperBuild/External_VTK.cmake

```diff
@@ -92,6 +92,11 @@ if((NOT DEFINED VTK_DIR OR NOT DEFINED VTK_SOURCE_DIR) AND NOT Slicer_USE_SYSTEM
       #-DVTK_USE_RPATH:BOOL=ON # Unused
       )
   endif()
+  if("${Slicer_VTK_VERSION_MAJOR}" GREATER_EQUAL "8")
+    if (CMAKE_CXX_STANDARD GREATER_EQUAL 11)
+      list(APPEND EXTERNAL_PROJECT_OPTIONAL_CMAKE_CACHE_ARGS -DVTK_USE_CXX11_FEATURES:BOOL=ON)
```

Comment:
I couldn't find reference to the variable `VTK_USE_CXX11_FEATURES` or the string `_USE_CXX11_FEATURES` in either VTK or CMake.